wire-ios/Wire-iOS/Sources/UserInterface/Conversation Options/CreatePasswordSecuredLink/CreateSecureGuestLinkViewController.swift

```diff
@@ -105,16 +206,71 @@ class CreateSecureGuestLinkViewController: UIViewController, CreatePasswordSecur
         viewModel.requestRandomPassword()
     }
 
+    // MARK: - Validation
+
+    func displayPasswordErrorState(for textFields: [UITextField], for labels: [UILabel]) {
+        for textField in textFields {
+            textField.textColor = validationErrorTextColor
+            textField.layer.borderColor = validationErrorTextColor.cgColor
+        }
+
+        for label in labels {
+            label.textColor = validationErrorTextColor
+        }
+
+    }
+
+    func resetPasswordDefaultState(for textFields: [UITextField], for labels: [UILabel]) {
+        for textField in textFields {
+            textField.applyStyle(.default)
+        }
+
+        for label in labels {
+            label.textColor = SemanticColors.Label.textFieldFloatingLabel
+        }
+
+    }
+
+    @objc
+    func handlePasswordValidation(for textField: ValidatedTextField) -> Bool {
```

Comment:
Do we need to return a value? This method is for textField, right?

Reply:
I don't think we need to return a value. I think the initial implementation was like that and then I forgot to remove it. Done!